linters/htmlhint.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+const htmlhint = require('htmlhint').HTMLHint;
+const { flatten, parseJson, readFiles, toArray } = require('../helper');
+
+const config = parseJson(__dirname + '/../config/htmlhint.hjson');
+
+// Add our custom rules.
+htmlhint.addRule({
+	id: 'doc-lang',
+	description: '<html> tags must have a "lang" attribute.',
+	init: configureDocLang
+});
+htmlhint.addRule({
+	id: 'link-href',
```

Comment:
In ESLint it's conventional for rules supplied by plugins to be prefixed with the plugin name, e.g. `jsdoc/check-param-names` or `jasmine/no-focused-tests`. This makes it easy to tell at a glance where to find the documentation for a particular rule. Would it be worth doing the same here?